app/services/slack/messengers/moderator_create_comment.rb

```diff
@@ -0,0 +1,42 @@
+module Slack
```

Comment:
Is this meant to be here?

Reply:
...I have no idea. Must be a git issue... Will update 😓

Reply:
Force pushed and resolved